### DISCUSSION TITLE
BUG

### DISCUSSION THREAD

**Discussion by Lu-7-M** [the1812/Bilibili-Evolved#2922] [提问 / Q&A] [answered]
因为B站视频播放器由于用了插件之后掩盖鼠标，我就把这个插件删了，但是删除后效果还在，整的我很头痛....

**Comment by Geeyun-JY3** [comment, marked as answer]
你这样啥信息没有，很难帮你排查问题。

另外，确认是本脚本的 bug 的话，请开 issue。按着模板填能提供有用信息。

**Reply by Lu-7-M** [reply]
https://github.com/the1812/Bilibili-Evolved/issues/2671
就是上面这种情况，我今天早上碰到的，弄了一上午没弄明白，被迫从Edge转用Chrome

**Reply by Geeyun-JY3** [reply]
那就去开 issue 啊，跟着模板走。
